src/components/ContextChart/index.js

```diff
@@ -79,6 +75,15 @@ class ContextChart extends Component {
     );
   };
 
+  getYScale = (s, height) => {
```

Comment:
Shouldn't this be calculated on the `Scaler` level?

Reply:
The way I see it, no. The ContextChart wants to have non-standard rendering, so it should shoulder that burden